docs/content/documentation/deployment/cloudflare-pages.md

```diff
@@ -71,5 +71,6 @@ From within the workers & pages dash, do the following:
 And add an environment variable `UNSTABLE_PRE_BUILD`, with the following value and save.
 
 ```sh
-asdf plugin add zola https://github.com/salasrod/asdf-zola && asdf install zola 0.18.0 && asdf global zola 0.18.0
+# Use zola latest binary version
```

Comment:
Just say: Replace <latest-version> by the current latest version of Zola

otherwise we can think it's some kind of feature from asdf (maybe it is? I don't know)